### PR TITLE
memory profiling, change connection mode to google storage

### PR DESCRIPTION
**Pull request by trilopin**
Change connection mode in Google Storage: now the connection is made every time is used though the Init() method (blank in fileStorage)

### CODE REVIEW

File: cmd/godinary/godinary.go [hunk 1]
```diff
@@ -74,20 +74,20 @@ func main() {
 	if viper.GetString("storage") == "gs" {
 		opts.GCEProject = viper.GetString("gce_project")
 		opts.GSBucket = viper.GetString("gs_bucket")
-		opts.GSCredencials = viper.GetString("gs_credentials")
+		opts.GSCredentials = viper.GetString("gs_credentials")
 		if opts.GCEProject == "" {
```
Comment (trilopin) [inline cmd/godinary/godinary.go:78]: just an old typo

File: cmd/godinary/godinary.go [hunk 2]
```diff
@@ -74,20 +74,20 @@ func main() {
...
 		}
-
-		opts.StorageDriver, err = storage.NewGoogleStorageDriver(opts.GCEProject, opts.GSBucket, opts.GSCredencials)
-		if err != nil {
-			log.Fatalf("can not create GoogleStorage Driver: %v", err)
+		opts.StorageDriver = &storage.GoogleStorageDriver{
+			BucketName:  opts.GSBucket,
+			ProjectName: opts.GCEProject,
+			Credentials: opts.GSCredentials,
 		}
```
Comment (trilopin) [inline cmd/godinary/godinary.go:91]: we don't need a constructor, use object instead

File: cmd/import/import.go [hunk 3]
```diff
@@ -78,5 +85,20 @@ func main() {
 		APIKey:    viper.GetString("cloudinary_apikey"),
 		APISecret: viper.GetString("cloudinary_apisecret"),
 	}
+
+	defer func() {
+		memProfile := viper.GetString("memprofile")
```
Comment (trilopin) [inline cmd/import/import.go:90]: write memory profiling stats if flag passed

File: http/views.go [hunk 4]
```diff
@@ -34,6 +34,13 @@ func Fetch(opts *ServerOpts) func(http.ResponseWriter, *http.Request) {
 		var reader io.ReadCloser
 		var dSem float64
 
+		err := opts.StorageDriver.Init()
+		if err != nil {
```
Comment (trilopin) [inline http/views.go:38]: force connection

**Review by miquelbar — APPROVED** [review]

**Pull request merged**